librabbitmq/amqp_openssl.c

```diff
@@ -662,8 +663,22 @@ destroy_openssl(void)
      * safely. We do leak the amqp_openssl_lockarray. Which is only
```

Comment:
I'd like to remember why we're intensionally leaking `amqp_openssl_lockarray`.

This code was originally patterned after what the postgres client-library does, which had a reason for doing what it does, but I'm not currently remembering what it was.